Vagrantfile

```diff
@@ -0,0 +1,33 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# Author: David Manouchehri
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "bento/ubuntu-16.04"
+
+	config.vm.synced_folder ".", "/vagrant", disabled: true
+
+	config.vm.provision "shell", inline: <<-SHELL
+		# apt-get update
+		# DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+		snap install --classic go
+		# apt-get clean
+	SHELL
+
+	config.vm.provision "docker"
+
+	config.vm.provision "shell", privileged: false, inline: <<-SHELL
+		go get -u github.com/linuxkit/linuxkit/src/cmd/linuxkit
```

Comment:
Seems a bit odd to reclone rather than reusing (or even sharing if such a thing is possible with Vagrant) the clone on the host where the Vagrant file is being run.

If that isn't possible then it may as well be in some subdirectory (if not some other repo altogether).

Reply:
I'm not too familiar with go, is there something like `go install ~/linuxkit` I could use? (I'm not sure if that handles dependencies, so I went with the safe bet of `go get [URL]`.)

Reply:
My point was you already have this exact repo cloned on the host, because you ran `vagrant up` from it, so fetching it again (whther with `go get`, or `go install` or whatever) inside the VM seems odd.

Reply:
I understood what you were saying in the first message, was explaining that dealing with golang's package dependencies wasn't trivial for someone who hasn't used it before.

Anyway, I figured it out. Done.